processing/src/main/java/org/apache/druid/query/aggregation/first/StringAggregatorUtils.java

```diff
@@ -29,7 +29,7 @@
 import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
 
-public class StringFirstLastUtils
+public class StringAggregatorUtils
```

Comment:
If you decide to end up moving `chop` to `StringUtils`, please revert this rename

Reply:
done